examples/resnet_inference_app.yaml

```diff
@@ -0,0 +1,53 @@
+name: resnet-inference
+
+resources:
+  cloud: aws
+  accelerators:
+    V100: 1
+
+inputs: {
```

Comment:
If we are forcing the cloud to be `aws`, can we remove the `inputs` and `outputs` field? The optimizer doesn't any options other than to use AWS in this YAML so these fields have no effect. If we want to keep them, we should remove `cloud: aws`

Reply:
Ah it does not need to be AWS (removed the field). And remove the inputs and outputs field.